CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ UNRELEASED
  * `kube` client connects to kubernetes via cluster dns when using `rustls` - #587 via #597
  * `kube` client now works with `rustls` feature in cluster - #153 via #597
  * `kube-core` added `CrdExtensions::crd_name` method (implemented by `kube-derive`) - #583
+ * TODO
```

Comment:
Starting a thread here so it's not forgotten

Reply:
I have added something now